server/sonar-web/src/main/js/app/integration/vsts/components/SonarCloudIcon.tsx

```diff
@@ -0,0 +1,39 @@
-.48-.54-.54-.48-.6-.4-.65-.32-.71-.24-.74-.14-.79-.05-.81.03-.65.09-.64.16-.62.22-.59.27-.56.32-.53.37-.5.41-.46.46-.42.5-.37.53-.33.57-.28.59-.22v-.09l.05-.81.15-.78.24-.75.32-.7.4-.66.48-.6.54-.54.6-.47.65-.41.71-.32.74-.24.78-.14L12 2l.81.05.78.14.75.24.7.32.65.41.61.47.54.54.47.6.4.66.32.7.24.75.15.78.05.81v.08l.05.01.27.09.27.11.26.12.26.13.25.14.25.15.23.16.23.17.23.18.21.19.2.2.2.21.19.22v.01l-.01-.01.2.26.19.27.17.28zM11.05 3.71l-.46.11-.43.16-.42.19-.39.23-.37.27-.34.29-.31.33-.28.36-.24.38-.21.41-.18.42-.13.45-.09.46-.04.48h.47l.14.01.15.01.15.01.14.01.15.02.14.02.14.02.14.03.14.03.13.03.14.04.13.04.13.04.12.04.13.05h.03l.07.03.07.03.06.02.07.03.07.03.07.03.06.04.07.03.06.03.07.04.06.04.06.04.06.04.06.04.06.04.05.04.04.03.03.03.03.03.02.04.03.03.02.04.02.04.02.04.02.04.01.04.02.04.01.04.01.04v.05l.01.04v.05l-.01.08-.01.09-.02.07-.03.08-.03.07-.04.07-.05.06-.05.06-.06.05-.06.05-.07.04-.07.03-.07.03-.08.02-.08.01-.09.01-.03-.01h-.08l-.03-.01-.04-.01H10l-.03-.01-.03-.02-.04-.01-.03-.01-.03-.02-.03-.02-.03-.02-.03-.02-.03-.02-.03-.02h.01l-.03-.01-.03-.02-.03-.02-.03-.02-.03-.02-.03-.02-.04-.01-.03-.02-.03-.01-.04-.02-.03-.01-.04-.02-.03-.01-.04-.02-.04-.01-.03-.01h-.02l.03.01-.03-.01-.07-.03-.1-.03-.1-.03-.11-.03-.1-.03-.1-.03-.11-.02-.11-.02-.1-.02-.11-.02-.11-.01-.11-.01-.11-.01-.11-.01H7.47l-.49.02-.48.08-.46.12-.45.16-.42.2-.4.25-.37.27-.35.32-.31.34-.28.37-.24.4-.21.43-.16.44-.12.46-.07.48-.03.5.03.49.07.49.12.46.16.44.21.43.24.4.28.37.31.34.35.31.37.28.4.25.42.2.45.16.46.12.48.08.49.02.26-.01.26-.02.25-.03.25-.05.24-.05.24-.07.23-.09.23-.09.22-.1.21-.12.21-.12.2-.14.2-.15.18-.15.18-.16.17-.17-.09-.16-.1-.19-.1-.19-.09-.2-.08-.19-.08-.21-.07-.2-.06-.21-.05-.21-.05-.22-.05-.21-.03-.22-.03-.22-.02-.23-.01-.22v-.32l.01-.08.02-.08.03-.07.04-.08.04-.06.04-.07.06-.05.05-.06.07-.04.06-.04.08-.04.07-.03.08-.02.08-.01H10.99l.08.01.07.02.08.03.07.04.07.04.06.04.06.06.05.05.05.07.04.06.03.08.03.07.02.08.01.08.01.08.02.5.08.48.12.46.16.45.2.42.25.4.28.37.31.35.34.31.37.28.4.24.43.2.44.17.47.12.48.07.49.03v-.02l.49-.03.48-.07.47-.12.44-.16.43-.21.4-.24.37-.28.34-.31.31-.34.28-.38.24-.4.21-.42.16-.45.12-.46.07-.48.03-.49-.02-.39-.04-.38-.07-.36-.1-.36-.13-.35-.15-.33-.18-.32-.19-.3-.23-.29-.24-.27-.26-.25-.28-.23-.29-.22-.32-.19-.32-.17-.35-.14-.01.05-.08.26-.09.26-.1.26-.12.25-.12.24-.13.24-.15.23-.15.23-.16.22-.17.21-.18.2-.19.2-.2.19-.2.18-.21.17-.22.17-.03.02-.02.01-.03.02-.03.02-.03.01-.03.01-.03.02-.03.01-.03.01-.03.01h-.03l-.03.01h-.04l-.03.01H15.18l-.08-.02-.08-.02-.07-.02-.08-.04-.06-.04-.07-.05-.05-.05-.06-.06-.04-.06-.04-.07-.04-.07-.02-.07-.02-.08-.02-.08v-.19l.01-.05.01-.05.02-.04.01-.05.02-.05.03-.04.02-.04.03-.04.03-.04.03-.04.03-.03.04-.04.03-.03.04-.03.02-.01.22-.17.22-.18.2-.2.19-.2.18-.22.16-.23.16-.24.14-.25.12-.25.11-.27.1-.27.07-.28.07-.28.04-.29.03-.3.01-.3-.03-.5-.07-.48-.12-.46-.16-.45-.21-.42-.24-.4-.28-.37-.31-.35-.34-.31-.38-.28-.4-.24-.42-.2-.44-.17-.47-.12-.48-.07-.49-.02-.49.02-.46.07zm-.03 14.36l-.02-.03v.01l.02.02z"
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Exceeds maximum line length of 180 ([typescript:S103](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS103))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3121&issues=AWILuukf54IFfP4XjVw2&open=AWILuukf54IFfP4XjVw2)

[ISSUE_KEY]: AWILuukf54IFfP4XjVw2